docs/quickstart.md

```diff
@@ -0,0 +1,157 @@
+# Quickstart Guide
+
+
+This guide covers how you can quickly get started using PMM. This is applicable to Docker compatible Linux based systems.
+
+
+## Prerequisites
+
+[Docker](https://docs.docker.com/engine/install/) 1.12.6 or higher.
+
+## System requirements
+
+**Disk**
+
+Approximately 1 GB of storage per monitored database node with data retention set to one week. 
+
+**Memory**
+
+A minimum of 2 GB per monitored database node. The increase in memory usage is not proportional to the number of nodes. For example, data from 20 nodes should be easily handled with 16 GB.
+
+**Ports**
+
+This is a list of [ports](setting-up/server/network.md) used by the various components of PMM.
+
+For PMM to work correctly, your system’s firewall should allow TCP traffic on these ports (UDP is not needed).
+
+## Install PMM
+
+!!! caution alert alert-warning "Important"
+    - You can use the [easy installation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/easy-install.html) script that will verify and install any missing software and dependencies. To use it, run the command with sudo privileges or as root.
+    - You can download the script from [github](https://github.com/percona/pmm/blob/main/get-pmm.sh).
+
+Here are the steps for installing the PMM server:
+
+1. Set up Docker on [Mac](https://docs.docker.com/docker-for-mac/install) or [Linux](https://docs.docker.com/install/linux/docker-ce/ubuntu).
+
+2. Install the PMM server using `cURL` or `wget` as follows:
+
+
+    === "cURL"
+
+        ```sh
+        curl -fsSL https://www.percona.com/get/pmm | /bin/bash
+        ```
+
+    === "wget"
+
+        ```sh
+        wget -qO - https://www.percona.com/get/pmm | /bin/bash    
+        ```
+
+3. Log in to PMM with the default login credentials provided after the installation is completed.
+
+For detailed instructions on installing the PMM server with the other methods, see [Setting up PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/index.html).
+
+## Add a database instance
+
+=== "MySQL 5.7 or 8.0"
+
+    Before you add a MySQL database for monitoring you should have a [database account for PMM](https://docs.percona.com/percona-monitoring-and-management/setting-up/client/mysql.html#create-a-database-account-for-pmm).
+
+    Here are the steps to add a MySQL database for monitoring:
+
+    1. From the PMM UI, go to **Configuration** → **PMM Inventory** → **Add Instance**.
+
+    2. Select **MySQL**. The **Add Service** page opens.
+
+    3. Enter your database credentials on the resulting page without changing any values.
```

Comment:
```suggestion
    2. On the **Add Service** page, enter your database credentials without changing any values.
```

Reply:
why we mention "without changing any values". I think user should know that he/she can change any values.